components/card/card_header_content.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+type Props = {
+    title: React.ReactNode;
+    subtitle?: React.ReactNode;
+    buttonText?: React.ReactNode;
+    onClick?: () => void;
+};
+
+const CardHeaderContent: React.FC<Props> = (props: Props) => {
```

Comment:
I would come up with a different name than `CardHeaderContent` here.
Maybe some like `AdminCardHeader` or `TitleAndButtonCardHeader`?

Reply:
Agreed. I came up with that name off top off my head but I believe Devin's `TitleAndButtonCardHeader` make more sense.